tasks.py

```diff
@@ -399,6 +399,11 @@ def database_exists():
     else:
         return True
 
+def trial():
```

Comment:
what is this task supposed to specifically do? Remove if doesn't any significance to the project

Reply:
@frankip  please ask a clearer question I can't understand it

Reply:
if the task doesn't add any significance to the project remove it